src/Library/demos/Carousel/main.blp

```diff
@@ -0,0 +1,76 @@
+using Gtk 4.0;
+using Adw 1;
+
+Box root_box {
+  orientation: vertical;
+	valign: center;
+	halign:center;
+  spacing: 10;
+
+	Adw.Carousel carousel {
```

Comment:
I think this needs the `spacing` property set, something like `12` or maybe `18`, otherwise it looks like the boxed list shadow is visible in the other pages.